api/app/signals/apps/api/v1/serializers.py

```diff
@@ -709,3 +709,11 @@ def create(self, validated_data):
         signal = Signal.actions.create_initial(
             validated_data, location_data, status_data, category_assignment_data, reporter_data)
         return signal
+
+
+class PrivateSignalNotInCategoriesSerializerList(HALSerializer):
```

Comment:
Isn't this just a SignalIdListSerializer? Bit easier to understand.

Reply:
agree (unless the plan is to extend this further)

Reply:
Changed, if we want to expand on this in the future we can always change it back